lib/gollum/public/gollum/javascript/editor/gollum.editor.js

```diff
@@ -55,7 +55,7 @@
       window.ace_editor = editor;
 
       editor.setTheme("ace/theme/tomorrow");
-      editor.setKeyboardHandler("ace/keyboard/vim");
+      editor.setKeyboardHandler("ace/keyboard/emacs");
```

Comment:
What is the reason for this change? I'm not against emacs instead of vim bindings, but if not necessary I would prefer to keep the old settings for continuity.